app/src/androidTest/java/ch/sdp/vibester/activity/GameSetupActivityTest.kt

```diff
@@ -41,13 +41,14 @@ class GameSetupActivityTest {
     }
 
     @Test
-    fun checkNothingSelect() {
-        onView(withId(R.id.nb_player_spinner)).check(matches(withSpinnerText("One")))
+    fun checkDefaultSelectDifficulty() {
```

Comment:
I feel it's more consistent to keep the same name for the test and the function under the test.